### PR TITLE
Install page updates for versions older than 'next'

### LINKED ISSUE #318: Install page: create from reworked download-and-build
Part of _New IA implementation_ #267:

- [x] `/docs/next/`:
  - [x] **Install** page from reworked `dl-build` page - #294, #323
  - [x] Rename file and add redirect rule - #319
- [ ] `/docs/v3.4/`, i.e., `/docs/latest`: #333 
  - [ ] **Install** page from reworked `dl-build` page
  - [ ] Rename file and add redirect rule
  - [ ] Add **Install** link to top-nav
- [ ] All other versions: #333
  - [ ] Rename all `dl-build` pages to `install`, so that the versions menu will work more smoothly
  - [ ] Change page titles to be **Install**
  - [ ] Add redirect rules

### PR DESCRIPTION
**Pull request by nate-double-u**
* Updating dl-bulid Install page from reworked next dl-build page
* Renaming file and adding redirect rule
* adding "Install" to top nav
* Renaming all dl_build pages to install, so that the versions menu will work more smoothly; adding redirect rules

fixes #318

### CODE REVIEW

**Comment by nate-double-u** [comment]
deploy previews:
* https://deploy-preview-333--etcd.netlify.app/
* https://deploy-preview-333--etcd.netlify.app/docs/v3.4/install/
* https://deploy-preview-333--etcd.netlify.app/docs/v3.3/install/
* https://deploy-preview-333--etcd.netlify.app/docs/v3.2/install/
* https://deploy-preview-333--etcd.netlify.app/docs/v3.1/install/

**Comment by nate-double-u** [comment]
Redirect checks:
* https://deploy-preview-333--etcd.netlify.app/docs/v3.3/dl_build
* https://deploy-preview-333--etcd.netlify.app/docs/v3.2/dl_build
* https://deploy-preview-333--etcd.netlify.app/docs/v3.1/dl_build

File: content/en/docs/v3.4/install.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: Install
+weight: 1150
+description: Instructions for installing etcd from pre-built binaries or from source.
+minGoVers: 1.16
```
Comment (chalin) [inline content/en/docs/v3.4/install.md:5]: This should be 1.13, right?

File: config.yaml [hunk 2]
```diff
@@ -180,3 +180,5 @@ menu:
       weight: -10
     - name: Blog
       url: /blog/
+    - name: Install
+      url: /docs/v3.4/install/
```
Comment (chalin) [inline config.yaml:184]: Add a #FIXME comment similar to the one in line 179

**Review by chalin — CHANGES_REQUESTED** [review]
See inline comments.

File: content/en/docs/v3.4/install.md (hunk 1 above)
Comment (nate-double-u) [reply content/en/docs/v3.4/install.md:5]: Correct, sorry good catch

**Comment by chalin** [comment]
Hmm, could you also change the page titles of v3.3, 3.2, 3.1 from **Download and build** to **Install**. (I know that wasn't part of the original #318 description. Thanks)

**Comment by nate-double-u** [comment]
I can do that, I had left them because there was no other content change.

**Review by chalin — APPROVED** [review]
LGTM (once the build finishes)

**Pull request merged**